spec/controllers/admin/spam_controller_spec.rb

```diff
@@ -16,7 +16,7 @@
       end
     end
 
-    %w[board communications translation tag_wrangling docs support open_doors].each do |role|
+    %w[board communications elections translation tag_wrangling docs support open_doors].each do |role|
```

Comment:
Should this and line 58 be `(Admin::VALID_ROLES - %w[superadmin policy_and_abuse]).each do |role|`?